### ISSUE TITLE
Volume

### ISSUE THREAD

**Issue by Luftloch80** [alex224/homebridge-openwebif-switch#3]
Possible to implement volume Control?

**Comment by alex224** [comment]
yes. i tried it and it seems to work. but i can see the volume slider only in the elgato eve app. the apple home app only shows the switch. maybe it currently doesn't support volume sliders.

**Comment by Luftloch80** [comment]
So Maybe just use a light Bulb as service and the Volume is the brigthness
